### ISSUE TITLE
[Images] Captions missing

### ISSUE THREAD

**Issue by evelyntorres1** [GSA/digitalgov.gov#1488]
## Issue

**Title** — **Results: 2014 Federal User Experience Survey**
**Demo URL** — https://demo.digital.gov//2014/11/21/results-2014-federal-user-experience-survey/
**Live URL** — https://digital.gov//2014/11/21/results-2014-federal-user-experience-survey/

---

**Images need captions

**Comment by greensteph** [comment]
_Corrected Links:_

Demo URL - [https://demo.digital.gov/2014/11/21/results-2014-federal-user-experience-survey/](https://demo.digital.gov/2014/11/21/results-2014-federal-user-experience-survey/)
Live URL - [https://digital.gov/2014/11/21/results-2014-federal-user-experience-survey/](https://digital.gov/2014/11/21/results-2014-federal-user-experience-survey/)
